packages/xsnap/test/test-xsnap.js

```diff
@@ -143,6 +143,30 @@ test('accept std regex range', async t => {
   await vat.terminate();
 });
 
+test('bigint map key', async t => {
+  const opts = options();
+  const vat = xsnap(opts);
+  t.teardown(() => vat.terminate());
```

Comment:
Since we’re using maker pattern and closures, this can be merely `t.teardown(vat.terminate)`.

Reply:
I tried something like that and lost one time recently. It might have been something that didn't use the maker pattern, so this might be a case of PTSD.